astro/src/content/docs/apis/_user-response-body-base.mdx

```diff
@@ -14,7 +14,7 @@ import UserTokenFieldsResponse from 'src/content/docs/apis/_user-token-response.
 import UserLoginFieldsResponse from 'src/content/docs/apis/_user-login-response.mdx';
 
 { (props.login_response || props.passwordless_login_response) &&
-<UserLoginFieldsResponse/>
+<UserLoginFieldsResponse {...props}/>
```

Comment:
The status quo appears to broken around this. `pendingIdPLinkId` for example is not returned on the identity provider responses according to the docs, but in reality it is.

This should fix that problem.